docs/apps/rabbitmq.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: rabbitmq
+title: RabbitMQ
+sidebar_label: RabbitMQ
+---
+
+Kubernetes operator to deploy and manage RabbitMQ clusters.
+
+## Overview
+
+[RabbitMQ](https://github.com/rabbitmq/cluster-operator) is a controller and custom resource definition (CRD) designed for the lifecycle (creation, upgrade, graceful shutdown) of a RabbitMQ cluster.
+
+## Creating RabbitMQ Clusters
+
+:::info ALERT
+Activate RabbitMQ App in the Apps Panel or ask your administrator.
+:::
+
+To create a RabbitMQ Cluster, on the team overview click on the `Catalog` in the sidebar and select the `RabbitMQ` workload. This will take you to the page with the readme where you can see all specifications and definitions that you can use to create your RabbitMQ Cluster with Queues and Policies.
+
+![Workload Catalog](../img/rabbitmq-1-click-workload.png)
+
+To configure the RabbitMQ Cluster workload go to the `Values` tab and fill in a name for the RabbitMQ Cluster workload. For example we created a rabbitMQ Cluster workload with the name `rabbit1` with 2 `queues` and 2 `policies`.
+
+![Add Name](../img/rabbitmq-2-add-name.png)
+![Fill In Values](../img/rabbitmq-3-add-queues-and-policies.png)
+
+:::info ALERT
+Do remember that even though these values can be edited afterwards, not all specifications or definitions can be updated after a `queue` or `policy` has been created. Please make sure everything is filled in correctly.
+:::
+
+When everything is filled in correctly you can `submit` and click the `deploy changes` button.
+
+To check the status deployment you can check the `tekton` pipeline and afterwards you can check the creation of the rabbitMQ pods and containers in ArgoCD.
```

Comment:
Although this is true, only admin user can see Tekton Otomi pipeline.
It would be better to rely solely on status from the workload list page.